doc/plugins/limit-count.md

```diff
@@ -37,7 +37,7 @@ Limit request rate by a fixed number of requests in a given time window.
 | -------------- | ------- | -------------------- | ------- | ------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | count          | integer | required             |         | [0,...]                                                                  | the specified number of requests threshold.                                                                                                                                                                                                                                                                 |
 | time_window    | integer | required             |         | [0,...]                                                                  | the time window in seconds before the request count is reset.                                                                                                                                                                                                                                               |
-| key            | string  | required             |         | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for"] | the user specified key to limit the rate.                                                                                                                                                                                                                                                                   |
+| key            | string  | required             |         | ["remote_addr", "server_addr", "http_x_real_ip", "http_x_forwarded_for", "consumer_name"] | to limit the number of requests. <br>For example, one can use the host name (or server zone) as the key so that we limit the number of requests per host name. Otherwise, we can also use the client address as a key, so that we can avoid a single client connecting to our service too many times. <br> Now accept those as key: "remote_addr"(client's IP), "server_addr"(server's IP), "X-Forwarded-For/X-Real-IP" in request header, "consumer_name"(consumer's username).                                                                                                                                                                                                                                                                   |
```

Comment:
I think the description of `”one can use the host name (or server zone) as the key so that we limit the number of requests per host name.“`  is unreasonable. Because in the `key` we support, except `consumer_name`, the ip address is obtained in other cases

Reply:
At present, my understanding of the project can only be translated into this way. Do you have any good suggestions? Thank you.

Reply:
I have a look at the description of the document. I feel that the field has been described in the front, so it is OK to write the currently supported key directly at the back

Reply:
Okay,I know.  if necessary, I think you can submit a `pr` to modify.

Reply:
Okay,I know. if necessary, I think can submit a `pr` to modify.

Reply:
> Okay,I know. if necessary, I think can submit a `pr` to modify.
OKay

Reply:
OKay